server/tests/services/datacommons_test.py

```diff
@@ -31,6 +32,38 @@ def get_json(filename):
     return json.load(f)
 
 
+class TestServiceDataCommonsV2Node(unittest.TestCase):
+
+  @mock.patch('server.services.datacommons.post')
+  def test_v2node_returns_response(self, mock_post):
+    response_with_next_token = get_json('v2node_response_with_next_token')
+
+    def side_effect(url, data):
+      assert url.endswith('/v2/node')
+      assert data == {
+          'nodes': ['dc/1', 'dc/2'],
+          'property': '->{property1,property2}',
+      }
+      return response_with_next_token
+
+    mock_post.side_effect = side_effect
+
+    self.assertEqual(v2node(['dc/1', 'dc/2'], '->{property1,property2}'),
+                     response_with_next_token)
+    assert mock_post.call_count == 1
+
+  @mock.patch('server.services.datacommons.post')
+  def test_large_input_size_makes_multiple_post_calls(self, mock_post):
+
+    input_dcids = [f'dc{i}' for i in range(10000)]
+    v2node(input_dcids, '->*')
+    assert mock_post.call_count == 19
+    print(mock_post.call_args)
```

Comment:
remove these print statements?

& maybe also check the response by creating a mini response for the first batch & a mini response for the second batch & check that the eventual response is right?